crates/sui-indexer/tests/json_rpc_tests.rs

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use sui_json_rpc_api::{CoinReadApiClient, IndexerApiClient, ReadApiClient};
+use sui_json_rpc_types::{
+    CoinPage, SuiObjectDataOptions, SuiObjectResponse, SuiObjectResponseQuery,
+};
+use sui_swarm_config::genesis_config::DEFAULT_GAS_AMOUNT;
+use test_cluster::TestClusterBuilder;
+
+#[tokio::test]
+async fn test_get_owned_objects() -> Result<(), anyhow::Error> {
+    let cluster = TestClusterBuilder::new()
+        .with_indexer_backed_rpc()
+        .build()
+        .await;
+
+    let http_client = cluster.rpc_client();
+    let address = cluster.get_address_0();
+
+    let objects = http_client
+        .get_owned_objects(
+            address,
+            Some(SuiObjectResponseQuery::new_with_options(
+                SuiObjectDataOptions::new(),
+            )),
+            None,
+            None,
+        )
+        .await?;
+    assert_eq!(5, objects.data.len());
+
+    // Multiget objectIDs test
+    let object_digests = objects
+        .data
+        .iter()
+        .map(|o| o.object().unwrap().object_id)
+        .collect();
+
+    let object_resp = http_client.multi_get_objects(object_digests, None).await?;
+    assert_eq!(5, object_resp.len());
```

Comment:
assert_eq on object_digests and digests from object_resp?

Reply:
I added an `assert_eq` comparing the results from fullnode and indexer, and actually found a discrepancy due to us not sorting by object ids. Also fixed in the latest commit.